lib/items/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :first_name, :last_name, :items
+  attr_reader :id
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1...1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
@Felipeg005 I think you missed a requirement in here.

- [ ]  should add self as a property of the item object (by using the correct setter from the item object)

I'd suggest you set the item property @author to be self in this method. You can do it now or in the next PR, just don't miss that part later.